### ISSUE TITLE
running mopidy as root doesn't allow it to use alsasink for audio output

### ISSUE THREAD

**Issue by frollic** [9and3r/mopidy-touchscreen#8]
If you set mopidy to run as root, you will get a warning in mopidy saying 
"you are using the SDL disk writer audio driver", and the file sdlaudio.raw
is created in /var/log/mopidy. 

root have to be added to the audio group for mopidy to work properly. 

> sudo usermod -a -G audio root

this is on raspbian.

**Comment by 9and3r** [comment]
Thanks for the tip. I will add it to the readme. 

Regarding sdlaudio.raw is related to:
`sdl_audiodriver = disk`

By default is set to disk as with some sound-cards you can have problems otherwise. 

The file generated will be empty but you can set `sdl_audiodriver = none` if you are not using a problematic sound card.

**Comment by frollic** [comment]
Hi,

thanks for the info.

I don't belive the sdl_audiodriver was set on my RPi, which would mean disk is the default value ? 
Also, the audio group might only be related to the kind of audio output used, in my case 
[audio]
output = alsasink

Not sure on how the sdlaudio.raw works, but if the file's constantly written to, i'äs probably a 
mem card killer on the RPi.

One more thing, I *think* I managed to get the sdlaudio.raw warning in mopidy even though
the whole [touchscreen] section had been removed from mopidy.conf, is that even possible ? 
:D

**Comment by 9and3r** [comment]
`I managed to get the sdlaudio.raw warning in mopidy even though
the whole [touchscreen] section had been removed from mopidy.conf, is that even possible ?`

Yes. The default config is [this](https://github.com/9and3r/mopidy-touchscreen/blob/develop/mopidy_touchscreen/ext.conf).
Any value you don't explicitly put on the config will be set with the value on  that file. Because of that if you don't specify disk is used.

##### What's the purpose of having `sdl_audiodriver = disk` value by default? #####
Mopidy-touchscreen uses [pygame](http://www.pygame.org/) for the UI.
Pygame not only allows to display grapichs, has also support of sounds. 
The problem is that in some soundcards [pygame takes exclusive use of the soundcard](https://discuss.mopidy.com/t/issue-with-gstreamer-music-not-playing-strange-error-message/324/9) at init and mopidy is not able to use the soundcard. 

As mopidy-touchscreen itself does not use sound capability of pygame I decided to use this disk driver, this way an empty file is created and the problem is gone. As mopidy-touchscreen has no sounds nothing will be writen to the file.

So in summary this value does not affect in any way the mopidy music and the output of the mopidy music is not written to the disk, but if you want to get rid of the warning set:

`sdl_audiodriver = none`

If you see that mopidy is not able to play after that, simply revert the line to disk again.
